### DISCUSSION TITLE
How's this different from the official version?

### DISCUSSION THREAD

**Discussion by ldpl-shrey** [formfcw/directus-extension-editable-layout#1] [Q&A] [answered]
It's a bit confusing, to see in the Marketplace 2 of the 'same' things, but by different teams. Don't know which one to continue with.

How's this one different from the 'official' one at https://github.com/directus-labs/extensions/tree/main/packages/spreadsheet-layout ?

And which one is actually the official one, as you have claimed in this repo that you created the 'official' Directus Labs one as well?

**Comment by formfcw** [comment, marked as answer]
Thanks for your great question. Actually, Editable Layout supports more field types/interfaces. While the goal of Spreadsheet Layout is to provide a spreadsheet-like experience, Editable Layout focuses on inline editing. I would not say that Spreadsheet Layout is the official one, but the original one. Editable Layout is based on it and will be developed further by me (and whoever wants to contribute). So the next versions of Editable Layout will allow more and more interfaces to be editable inline in table view. The decision which one to use is up to you. Actually, it's no problem to use both. 🤝
